deny.toml

```diff
@@ -0,0 +1,113 @@
+[sources.allow-org]
+github = [
+    "mintlayer", #allow any code from mintlayer's github
+]
+
+[bans]
+# We only use cargo deny to validate depdendency licenses.
+# The duplicate-crate validation is performed in 'codecheck.py'.
+multiple-versions = "allow"
+deny = []
+
+#skip quick-error for "reasons" that i cba to explain just about as much as i cba to fix
+skip = [
```

Comment:
This list is why I abandoned cargo deny. If we have to keep this list maintained, we won't add cargo deny. It's too difficult to maintain on updates.

Reply:
We don't need most of that stuff. There's nothing wrong with using cargo deny for this as the plan is to use it purely to catch gpl stuff. The issue here is that the deny.toml file has things that we don't need here anymore

Reply:
Everything but the licenses (including the licenses.clarify) stuff can go

Reply:
I'm OK also with keeping the advisories

Reply:
Thanks for the comments. I only kept the license and advisories stuff in the deny.toml file.
Btw, now it is only checking the dependency licenses. For adding advisories in the same GitHub job, the next could be used but only considering 'licenses' and 'advisories' checks:

https://github.com/marketplace/actions/cargo-deny#recommended-pipeline-to-avoid-sudden-breakages